lib/datadog.py

```diff
@@ -285,7 +286,7 @@ def compile(install_path, cache_dir):
     # aws s3 cp ../dd-vX.Y.Z.tar.gz s3://mx-cdn/mx-buildpack/experimental/
     buildpackutil.download_and_unpack(
         buildpackutil.get_blobstore_url(
-            "/mx-buildpack/experimental/dd-v0.9.0.tar.gz"
+            "/mx-buildpack/experimental/{}".format(DD_SIDECAR)
```

Comment:
Do we have any recovery in place if this file does not exist on the CDN?

Reply:
No, not yet. It's just a version bump now.

Reply:
Seems a little risky; since I presume deploys will fail if we forget to upload this file. Is there a unit test that checks this?